FBSimulatorControl/Management/FBSimulatorControl.m

```diff
@@ -115,4 +115,16 @@ - (BOOL)firstRunPreconditionsWithError:(NSError **)error
   return YES;
 }
 
++ (FBSimulatorPool *)poolForConfiguration:(FBSimulatorControlConfiguration *)configuration
+{
+  return [FBSimulatorPool poolWithConfiguration:configuration deviceSet:[self deviceSetForConfiguration:configuration]];
```

Comment:
Why do you need to pass device set which is taken from configuration which is already passed as an argument? Just read it from configuration in constructor?